backend/src/test/java/com/yigongil/backend/acceptance/steps/StudyProgressStep.java

```diff
@@ -0,0 +1,52 @@
+package com.yigongil.backend.acceptance.steps;
+
+import com.yigongil.backend.response.StudyDetailResponse;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.springframework.http.HttpHeaders;
+
+import java.time.Duration;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class StudyProgressStep {
+
+    private final SharedContext sharedContext;
+
+    public StudyProgressStep(SharedContext sharedContext) {
+        this.sharedContext = sharedContext;
+    }
+
+    @Given("1일이 지난다.")
+    public void 시간_소요() throws InterruptedException {
+        Thread.sleep(Duration.ofSeconds(2).toMillis());
```

Comment:
p2: 물론 @Given안의 문자열이 명확하지만 나중에 보면 헷갈릴수도 있으니 매직넘버에도 이름을 설정해주어도 좋을 것 같네요
```suggestion
        Thread.sleep(Duration.ofSeconds(ONE_DAY).toMillis());
```